modules/core/src/main/java/org/apache/ignite/dump/DumpReaderConfiguration.java

```diff
@@ -69,26 +69,6 @@ public DumpReaderConfiguration(File dir, DumpConsumer cnsmr) {
         this(dir, cnsmr, DFLT_THREAD_CNT, DFLT_TIMEOUT, true, true, null, false);
     }
 
-    /**
```

Comment:
Why this change?

Reply:
This is a cleanup of unused code left from previous ticket.